practice_21_03_2019/Program.cs

```diff
@@ -45,5 +49,35 @@ public static void CreateMixedArray()
                 Console.Write($"{mixedArray[i]} ");
             }
         }
+
+        public static void ArrayMaxValue()
+        {
+            int[] randomArr = new int[13];
+            Random randomGenerator = new Random();
+            for (int i = 0; i < 13; i++)
```

Comment:
I would suggest to use randomArr.Length instead of integer literal - 13 in **for** operator